tools/ci_build/github/apple/objectivec/onnxruntime-mobile-objc.podspec.template

```diff
@@ -29,6 +29,7 @@ Pod::Spec.new do |s|
 
     core.pod_target_xcconfig = {
       "HEADER_SEARCH_PATHS" => include_dirs.join(" "),
+      "OTHER_CPLUSPLUSFLAGS" => "-fvisibility=hidden -fvisibility-inlines-hidden",
       "EXCLUDED_ARCHS[sdk=iphonesimulator*]": "arm64",
```

Comment:
`"EXCLUDED_ARCHS[sdk=iphonesimulator*]": "arm64"` just noticed the inconsistent syntax, ":" vs "=>"
I guess it's not invalid

Reply:
Fixed, but this should not matter since the c package already specified it and c package is the dependency anyway